adr/0016-discarded-discovery-events.md

```diff
@@ -0,0 +1,36 @@
+# 16. Discarded Discovery Events
+
+Date: 2024-10-15
+
+## Status
+
+Accepted
+
+## Context
+
+The discovery workflow allows Trento Agent instances to publish their host system's information to the Trento Web's `collect` API operation. Once accepted, the event results in one or more commands that are dispatched to the ES/CQRS system.
+
+There are cases in which a discovery event sent from a Trento Agent is not processed; they can be grouped as:
```

Comment:
```suggestion
There are cases in which a discovery sent from a Trento Agent is not processed; they can be categorized as:
```

Reply:
I like the rephrase. Do you want to remove the first sentence, too, or it's just a typo?

Reply:
My suggestion is to omit that, because _it is known what a discovery is_.
However feel free to keep it, if it helps as an extra context. It does not harm anyway.

Reply:
I'd rather keep it to give a bit of context to occasional readers. It's just a sentence and it brings no much noise IMHO